src/templates/build-page/tbtc-sdk/FeaturesSection.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC } from "react"
+import { SimpleGrid, Stack } from "@chakra-ui/react"
+import Card from "../../../components/Card"
+import { BodyMd, H6, Image, ImageProps } from "../../../components"
+
+export interface FeaturesSectionProps {
+  cards: FeatureCardProps[]
+  icon: ImageProps
+}
+
+export interface FeatureCardProps {
+  title: string
+  description: string
+  isBigSize?: boolean
+}
```

Comment:
Unnecessary exports. We use those interfaces only in this file.

```suggestion
interface FeaturesSectionProps {
  cards: FeatureCardProps[]
  icon: ImageProps
}

interface FeatureCardProps {
  title: string
  description: string
  isBigSize?: boolean
}
```

Reply:
Addressed in https://github.com/threshold-network/website/pull/105/commits/02522d97d220ee5e52ee32525e9d80228eec79d7